x-pack/plugins/licensing/common/license.ts

```diff
@@ -169,10 +183,13 @@ export class License implements ILicense {
   }
 
   getFeature(name: string) {
-    if (!this.featuresMap.has(name)) {
-      this.featuresMap.set(name, new LicenseFeature(name, this.features[name], this));
+    let feature = this.featuresMap.get(name);
```

Comment:
could you add an integration test to see how service API will be used by plugins?

Reply:
Integration will be handled in a separate PR, there's another issue created for it.